storage/innobase/dict/dict0dict.cc

```diff
@@ -4645,6 +4646,7 @@ dict_foreign_parse_drop_constraints(
 		ptr1 = dict_accept(cs, ptr1, "EXISTS", &success);
 		if (success) {
 			ptr = ptr1;
+			is_exists = true;
```

Comment:
`if_exists` would be a more descriptive name, because the syntax is `IF EXISTS`.